ui/src/app/views/workspace-edit/component.spec.ts

```diff
@@ -1,52 +1,28 @@
-import {ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
```

Comment:
Should this move to being a tsx file? I believe you mentioned still working on tests?